cmd/userPromote.go

```diff
@@ -2,7 +2,9 @@ package cmd
 
 import (
 	"github.com/netsage-project/grafana-dashboard-manager/api"
+	"github.com/netsage-project/grafana-dashboard-manager/config"
 	"github.com/sirupsen/logrus"
+	log "github.com/sirupsen/logrus"
```

Comment:
This looks like a duplicate import. `log "github.com/sirupsen/logrus"` seems to be the more common pattern

Reply:
Fixed.